src/rules/consistent-callback-naming.ts

```
@@ -0,0 +1,82 @@
+                return fixer.replaceText(node.name, `on${eventName}`);
+              },
+            });
+          }
+        }
+      },
+
+      // Check function declarations and variable declarations for callback functions
+      'FunctionDeclaration, VariableDeclarator'(node: any) {
+        const functionName = node.id?.name;
+        
+        if (functionName && functionName.startsWith('handle')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)